docs/debug.md

```diff
@@ -33,7 +33,7 @@ You can customize the core dump pattern with the following options
 
 There is a debug snap that can be installed with:
 
-    $ sudo snappy install snappy-debug
+    $ sudo snap install snappy-debug
```

Comment:
I'll just leave the note here that this is not yet working:
https://bugs.launchpad.net/snappy/+bug/1543118
I don't think it's necessary to remove it from the docs, at least not yet because I hope we'll get it back soon.